### PR TITLE
Akif_JS2W1_Homework

### PR DESCRIPTION
**Pull request by Akif2709**

### CODE REVIEW

File: Week1/homework/app.js [hunk 1]
```diff
@@ -1,11 +1,122 @@
...
+  'leo_tolstoy_resurrection',
+];
+
+function generateList() {
+  const createUl = document.createElement('ul');
+  document.body.appendChild(createUl);
+  for (let i = 0; i < bookTitles.length; i++) {
+    const createLi = document.createElement('li');
+    createUl.appendChild(createLi);
+    createLi.setAttribute('id', `${bookTitles[i]}`);
```
Comment (paulbremer) [inline Week1/homework/app.js]: The setAttribute accepts a string, `bookTitles[i]` already returns a string from the array so there is no need the put it inside a string. 

`createLi.setAttribute('id', bookTitles[i]);`

File: Week1/homework/app.js [hunk 2]
```diff
@@ -1,11 +1,122 @@
...
+  turgenev_fathers_and_sons: 'images/fathers.jpg',
+  leo_tolstoy_resurrection: 'images/leo.jpg',
+};
+
+function generateCover() {
+  for (const cover of Object.keys(bookCovers)) {
+    const createImg = document.createElement('img');
+    const list = document.getElementById(cover);
+    list.appendChild(createImg);
+    createImg.setAttribute('src', `${bookCovers[cover]}`);
```
Comment (paulbremer) [inline Week1/homework/app.js]: See previous comment.

File: Week1/homework/app.js [hunk 3]
```diff
@@ -1,11 +1,122 @@
...
+    createImg.setAttribute('alt', `${cover}`);
+  }
+
+  for (let i = 0; i < bookTitles.length; i++) {
+    const createHeading = document.createElement('h2');
+    document.getElementById(`${bookTitles[i]}`).appendChild(createHeading);
```
Comment (paulbremer) [inline Week1/homework/app.js]: Same here, `document.getElementById(bookTitles[i]).appendChild(createHeading);`

File: Week1/homework/style.css [hunk 4]
```diff
@@ -1 +1,71 @@
...
+img {
+  max-width: 100%;
+  position: block;
+  width: 250px;
+  height: 80%;
+  box-shadow: -2px -2px 6px white;
+}
+
+ul {
+  display: flex;
```
Comment (paulbremer) [inline Week1/homework/style.css]: Very nice use of `flex`!

File: Week1/homework/style.css [hunk 5]
```diff
@@ -1 +1,71 @@
...
+  border-radius: 3%;
+  opacity: 0.9;
+}
+
+p {
+  font-size: 1.2em;
+  font-family: cursive;
+}
+
+@media (max-width: 1200px) and (min-width: 800px) {
```
Comment (paulbremer) [inline Week1/homework/style.css]: Bonuspoints for responsive design!

**Comment by Akif2709** [comment]
Hello Paul, I have made corrections as you say. Thanks for review.

**Pull request closed without merging**